src/Microsoft.AspNetCore.Razor.Design/project.json

```diff
@@ -36,9 +36,9 @@
     }
   },
   "dependencies": {
-    "Microsoft.AspNetCore.Razor.Runtime": "1.1.0-*",
-    "Microsoft.Extensions.CommandLineUtils": "1.1.0-*",
-    "Microsoft.Extensions.DependencyInjection": "1.1.0-*",
+    "Microsoft.AspNetCore.Razor.Runtime": "1.0.0-*",
+    "Microsoft.Extensions.CommandLineUtils": "1.0.0-*",
+    "Microsoft.Extensions.DependencyInjection": "1.0.0-*",
```

Comment:
The design package will bump up the version of `Microsoft.NETCore.App` to `1.1.0-*`. So we'd have to downgrade that to the LTS build `1.0.0`. Additionally the tool would also require to target the 1.0.0 version of the `M.NETCore.App` since that would be the LTS version of the shared runtime installed on user's machines.